### ISSUE TITLE
Explicit relative path for `MD057`

### ISSUE THREAD

**Issue by martimlobao** [rvben/rumdl#537]
### Problem or motivation

I'm not sure if I changed some setting unwittingly or if this is some recent behavior change, but I recently started getting a bunch of warnings for `MD057` in Obsidian which I wasn't getting before. `rumdl` is unable to find the right file because Obsidian defaults to a configured path for attachments, so `rumdl` is not able to find the file without knowing what the root is.

### Proposed solution

`MD057` almost has this functionality with `absolute-links = "relative_to_docs"`, but the path is inferred from `docs_dir` in `mkdocs.yml`, which isn't applicable here. Would it make sense to add an option to specify a more generic root path instead? Don't know if it should be a new option or just generalize the existing option to e.g.

```toml
[MD057]
absolute-links = "relative"
relative-path = "Attachments"

### Alternatives considered

_No response_

### Feature category

Improve existing rule

### Example markdown

```markdown
[MD057]
absolute-links = "relative"
relative-path = "Attachments"
```

### Contribution

- [ ] I would be willing to submit a PR to implement this feature

**Comment by rvben** [comment]
This functionality already exists as of v0.1.56:

1. **Obsidian auto-detection**: Set `flavor = "obsidian"` in `[global]` and rumdl will automatically read `.obsidian/app.json` to resolve `attachmentFolderPath` (supports all 4 Obsidian modes: vault root, named folder, same as file, subfolder).

2. **Explicit search paths**: You can also configure additional directories manually:
   ```toml
   [MD057]
   search-paths = ["Attachments"]
   ```

Either approach should resolve the false positives you're seeing. Let me know if you run into any issues after upgrading.

**Comment by martimlobao** [comment]
Thanks @rvben, this works perfectly!

**Comment by rvben** [comment]
Thanks @martimlobao for the confirmation! 🚀
